### ISSUE TITLE
Multiple log messages "No oshi.properties file found from ClassLoader ..."

### ISSUE THREAD

**Issue by notOM3GA** [oshi/oshi#1619]
I have shaded oshi into a program that is loaded by a different ClassLoader than the internal class loader.

Whenever I do anything with oshi, it spits this error twice.

> No oshi.properties file found from ClassLoader jdk.internal.loader.ClassLoaders$AppClassLoader@446cdf90

**Comment by mprins** [comment]
have you shaded in the properties file as well? (there are in fact several property files that you need)

**Comment by dbwiddis** [comment]
OSHI attempts to load properties [three different ways](https://github.com/oshi/oshi/blob/15164b832a9706e58f81734a32bb2c9a84f78226/oshi-core/src/main/java/oshi/util/FileUtil.java#L236-L238); if the first one fails it falls back to the second, and then the third.  You appear to be failing in the first two attempts which is not unexpected given your use of a different classloader.  See #1285 fixed by #1287.

These are `INFO` level logging and perhaps should be `DEBUG` as they are normal fallback attempts.  Only if you ultimately receive three failure attempts followed the `WARN` message "Failed to load default configuration" is there a true concern.

Bottom line: you may safely ignore these messages; and we should change the log level to DEBUG.

**Comment by dbwiddis** [comment]
### First Timers Only

I'm marking this issue for [first-timers-only](https://www.firsttimersonly.com/).  That means that I will only accept a PR for this one from someone who's never contributed to an open source project on GitHub before.

The main goal of this task is to learn the process for submitting a PR -- the GitHub equivalent of writing a "Hello World!" program.  If you already know the PR workflow, please leave this issue for someone else!

This one is easy (but don't make that statement make you feel bad if you have a hard time with it, there's more to contributing to open source than changing lines of code, especially if it's your first time). I'll hold your hand through this if you need me to. 😄 

### The improvement you are making

In #1287, we added the ability to load OSHI's property files from other than the system ClassLoader. There are three attempts, leading to two `INFO` level log messages even if the third one is successful, but giving an apparent indication of a failure.  These log messages should be changed to `DEBUG` level to reduce confusion.

### What to do 

In [FileUtil.java](https://github.com/oshi/oshi/blob/15164b832a9706e58f81734a32bb2c9a84f78226/oshi-core/src/main/java/oshi/util/FileUtil.java#L255) at line 255, change `LOG.info()` to `LOG.debug()`.

### How to submit your change to the repository

Here are the steps to get a PR merged here.
- Read the [CONTRIBUTING](https://github.com/oshi/oshi/blob/master/CONTRIBUTING.md) file for reference.  We'll follow those steps, and I'll walk you through anything you don't understand.
- Fork the project to your own GitHub account, make a local clone, and create a branch for your submission.
- Make the changes described above.
- Commit your changes, push to to your GitHub fork, and create a Pull Request
- Respond to the usually minor comments on code review
- Celebrate! :tada:

**Comment by dbwiddis** [comment]
In addition to the log level change, I see there was some wisdom in @ngyukman initially collecting the various classloaders in a set, which would avoid the redundancy of the first two attempts: see https://github.com/oshi/oshi/pull/1287/commits/8d690b42427e2184c0b66086c39108373f115dd6

Would be nice to also put that back in, except ensuring order is preserved using `.collect(Collectors.toCollection(LinkedHashSet::new))`.

**Comment by dbwiddis** [comment]
Log level fixed in #1620. Putting in the redundant message removal in #1621.
